pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -81,6 +81,14 @@ type launchTemplateOptions struct {
 	SecurityGroupsIds []string
 	AMIID             string
 	Tags              map[string]string
+	MetadataOptions   launchTemplateMetadataOptions
+}
+
+type launchTemplateMetadataOptions struct {
```

Comment:
Why do we need a new type for this? Can't we use the v1alpha1.MetadataOptions type?